data.sql

```diff
@@ -13,4 +13,44 @@ INSERT INTO animals VALUES  (1, 'Agumon', 'Feb 3, 2020', 0, 'true', 10.23 ),
                             (9, 'Boarmon', 'Jun 7, 2005', 7, 'true', 20.4),
                             (10, 'Blossom', 'Oct 13, 1998', 3, 'true', 17),
                             (11, 'Ditto', 'May 14, 2022', 4, 'true', 22);
-                            
\ No newline at end of file
+
+______________________________________________________________________________________________________________________________
+
+
+INSERT INTO owners (full_name, age) VALUES ('Sam Smith', 34), ('Jennifer Orwell', 19), ('Bob', 45), ('Melody', 77), ('Dean Winchester', 14), ('Jodie Whittaker', 38);
+
```

Comment:
You use have done a great job on minifying your code by entering all the data to be entered in one ```INSERT INTO``` statement instead of writing a new ```INSERT INTO``` statement for each of the entries 👍. However, I believe that in the 4th entry, the data to be entered was ```('Melody Pond', 77)```. You left out 'Pond' in the full name 😅. I know this is a small change and might even seem trivial but I encourage you to build the habit of paying rapt attention to detail as this is a skill that would greatly help you when you are preparing your projects for job interviews. 😉